library/bignum.c

```diff
@@ -1018,7 +1018,7 @@ int mbedtls_mpi_sub_mpi( mbedtls_mpi *X, const mbedtls_mpi *A, const mbedtls_mpi
     MPI_VALIDATE_RET( B != NULL );
 
     s = A->s;
-    if( A->s * B->s > 0 )
+    if( !( A->s ^ B->s ) )
```

Comment:
As above, this is a fancy way of writing `A->s == B->s`.

Reply:
As above: ...and how excessive is `A->s * B->s > 0` to do the same?
:rocket: